code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -22,11 +21,15 @@
 	return ..()
 
 /datum/action/innate/cult/blood_magic/proc/Positioning()
+	var/list/screen_loc_split = splittext(button.screen_loc,",")
+	var/list/screen_loc_X = splittext(screen_loc_split[1],":")
+	var/list/screen_loc_Y = splittext(screen_loc_split[2],":")
+	var/pix_X = text2num(screen_loc_X[2])
 	for(var/datum/action/innate/cult/blood_spell/B in spells)
-		var/pos = -29+spells.Find(B)*31
-		B.button.screen_loc = "6:[pos],4:-2"
-		B.button.moved = B.button.screen_loc
-		B.button.locked = TRUE
+		if(B.button.locked)
```

Comment:
These really should be dedicated obj/screen elements instead of action buttons if you want a preset positions.

Reply:
You mean like alerts? I need an example to understand what your going for here because I mean the system was already merged as action buttons, this just improves it, and hopefully after this PR nobody really has to ever fuss about where it is on their screen ever again.